src/api/order/services/post.js

```diff
@@ -0,0 +1,30 @@
+import Order from '../../../models/order';
+import { sendSuccessResponse, sendErrorResponse } from '../../../utils/responseHelpers';
+
+async function createOrder(req, res) {
+  const { productList } = req.body;
+  const { orderProducts, user } = req;
+  const order = { userId: user._id, productList: [] };
+  const updatedProducts = [];
+  let allPrice = 0;
+  let quantityOrderItem;
```

Comment:
Why is quantityOrderItem here? You use this only inside of your loop, so why you declared it here?